packages/phrasebook/src/index.ts

```diff
@@ -5,6 +5,9 @@ export type {
   Translations
 };
 
+// eslint-disable-next-line no-console
```

Comment:
did you test something? Let's remove this.

Reply:
Removed